exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -88,6 +89,7 @@
   String USER_AUTHENTICATION_ENABLED = "drill.exec.security.user.auth.enabled";
   String USER_AUTHENTICATOR_IMPL = "drill.exec.security.user.auth.impl";
   String PAM_AUTHENTICATOR_PROFILES = "drill.exec.security.user.auth.pam_profiles";
+  String USER_DELEGATION_ENABLED = "drill.exec.delegation.enabled";
```

Comment:
Isn't an empty delegation block enough? Any reason to have a second kill switch?